help/whats_new/_posts/2019-02-01-whats-new.md

```diff
@@ -3,6 +3,69 @@ This page is updated monthly with a list of new (**[new]**) and improved (**[imp
 
 To receive this listing via monthly email newsletter, along with announcements and new blog posts, [sign up here](http://eepurl.com/ggoxhn).
 
+## March 2024
+* Announcements:
+   + <b>Applications Now Being Accepted for the NASA SciX Ambassador Program</b> - Ambassadors will serve as NASA SciX user community representatives, actively engaging with the scientific community across various disciplines within NASA's Science Mission Directorate. They will work with the NASA SciX team to foster connections among researchers in fields such as Astrophysics, Earth Science, Planetary Science, Heliophysics, and Biological & Physical Sciences. As communicators, ambassadors will convey information about the NASA SciX digital library resource, expanding NASA SciX access and impact for new audiences. Their leadership will drive discussions, organize events, and provide mentorship to fellow researchers, contributing to the advancement of open science. In recognition of their contributions, ambassadors will receive community outreach training, visibility and recognition for their contributions as a NASA SciX community leader, and financial support to attend in-person trainings at the Harvard & Smithsonian Center for Astrophysics and other conferences where they will present on NASA SciX and their research. We invite you to [learn more about the program and apply here](https://docs.google.com/forms/d/e/1FAIpQLSeMmhfklhEDl7sc2uP9Tk7hSQrSjHiTBwL42FIeuYB5L4dOBA/viewform) by 4 April 2024. 
+
+   + <b>NASA SciX Team Presentations at the Year of Open Science Online Conference</b> - Please join members of the NASA SciX team at the upcoming Year of Open Science (YOS) Culminating Conference taking place online March 21 and 22, 2024. Hosted by the Center for Open Science in collaboration with NASA, this free event will showcase outcomes, coalition-building efforts, and ongoing work stemming from the 2023 Year of Open Science (YOS). [Learn more and register here.](https://www.cos.io/yos-conference)
+
+   + <b><i>[New Blog Post!](../../blog/userfeedback-2024)</i> Highlights the Role of User Feedback in the ADS to NASA SciX Expansion</b> - The crucial role of user feedback in shaping the evolution of both ADS and the NASA Science Explorer (SciX) is the focus of a new blog post. Explore how your input is driving enhancements to search functionality and ensuring a positive transition from ADS to the expanded NASA SciX platform [here](../../blog/userfeedback-2024).
+ 
+   + <b>Assessing your Observatory's Impact: Best Practices in Establishing and Maintaining Observatory Bibliographies</b> - A new publication from the Observatory Bibliographers Collaboration features the use of the Astrophysics Data System as a best practice in establishing and maintaining bibliographies for observatories wishing to measure and evaluate the scientific output and overall impact of their facilities. Read the open access article [here](https://arxiv.org/abs/2401.00060). 
```

Comment:
I'd include a link to ADS somewhere in here too: https://ui.adsabs.harvard.edu/abs/2024arXiv240100060O/abstract